todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskFragment.java

```diff
@@ -30,23 +28,38 @@
 import android.view.ViewGroup;
 import android.widget.TextView;
 
+import com.example.android.architecture.blueprints.todoapp.Injection;
 import com.example.android.architecture.blueprints.todoapp.R;
+import com.example.android.architecture.blueprints.todoapp.data.Task;
+import com.google.common.base.Preconditions;
+
+import rx.android.schedulers.AndroidSchedulers;
+import rx.functions.Action1;
+import rx.schedulers.Schedulers;
+import rx.subscriptions.CompositeSubscription;
 
 /**
  * Main UI for the add task screen. Users can enter a task title and description.
  */
-public class AddEditTaskFragment extends Fragment implements AddEditTaskContract.View {
+public class AddEditTaskFragment extends Fragment {
 
     public static final String ARGUMENT_EDIT_TASK_ID = "EDIT_TASK_ID";
 
-    private AddEditTaskContract.Presenter mPresenter;
-
+    @Nullable
     private TextView mTitle;
 
+    @Nullable
     private TextView mDescription;
 
+    @Nullable
     private String mEditedTaskId;
 
+    @Nullable
+    private CompositeSubscription mSubscription;
+
+    @Nullable
+    private AddEditTaskViewModel mViewModel;
+
     public static AddEditTaskFragment newInstance() {
         return new AddEditTaskFragment();
     }
```

Comment:
The thing about next line ``public AddEditTaskFragment()`` as I cannot put the comment there. Do you really need to explicitly create that ctor?